# Copyright 2022 Ellis Ratner (eratner@berkeley.edu)

# Pose of the robot where the execution error occured.
float64 x
float64 y
float64 yaw

# Action the robot was taking when execution error occured.
float64 action_dir_x
float64 action_dir_y
float64 action_dir_yaw

ellis_planner/ControlHistory control_history

---
bool success